src/main/java/org/teamresistance/frc/UltraSensor.java

```diff
@@ -0,0 +1,35 @@
+package org.teamresistance.frc;
+
+import org.strongback.command.Command;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * @author Tarik C. Brown
+ */
+public class UltraSensor extends Command {
+  public Ultrasonic ping1;
+  public Ultrasonic ping2;
+  public double range;
+  public double range2;
+  public UltraSensor() {
+    // Should be Ultrasonic(Ping Channel, Echo Channel
+    // Sets the channels and makes the sensors start running in auto
+    ping1 = new Ultrasonic(0,1);
```

Comment:
Strongback has a nice wrapper around WPILib's `Ultrasonic` class that handles the call to `setAutomaticMode` for us and includes a built-in soft `zero` method. The `DistanceSensor` API also keeps things strictly read-only. Could you migrate over to Strongback's wrapper as you're moving things to `IO`?

**WPILib**:
```java
// Setting up
Ultrasonic sensor = new Ultrasonic(0, 1);
sensor.setAutomaticMode(true);

// Reading distance
double distance = sensor.getRangeInches();
```

**Strongback:**
```java
// Setting up
DistanceSensor sensor = Hardware.DistanceSensors.digitalUltrasonic(0, 1);

// Reading distance
double distance = sensor.getDistanceInInches();
```